hardhat.config.ts

```diff
@@ -5,30 +5,42 @@ import './utils/assert'
 
 dotenv.config()
 
+const MAINNET_RPC_URL = process.env.RPC_URL
+const GOERLI_RPC_URL = process.env.GOERLI_RPC_URL
+
+if (!MAINNET_RPC_URL && !GOERLI_RPC_URL) {
+  throw new Error(`RPC url was not provided. Please, ensure the .env file is filled correctly.`)
+}
+
 const WALLET_PRIVATE_KEY = process.env.WALLET_PRIVATE_KEY!
```

Comment:
:grey_question: maybe have `MAINNET_WALLET_PRIVATE_KEY` and `GOERLI_WALLET_PRIVATE_KEY` to discourage using the same PK on real and test networks (not a strong opinion, though, posting separately)